app/src/main/java/com/easyfitness/DAO/bodymeasures/BodyPartExtensions.java

```diff
@@ -32,7 +32,8 @@
     public static final int TRAPEZIUS = 24;
     public static final int OBLIQUES = 25;
     public static final int SHOULDERS = 26;
-    public static final int SIZE = 27;
+    public static final int GLUTES = 27;
```

Comment:
You cannot assign Glutes to an existing number (27). It will mess up the database for existing users. You probably didn't see it in your test because you made a fresh install.
So SIZE should stay at 27 and GLUTES should be 28.